core/crypto/hasher/hasher_impl.cpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "crypto/hasher/hasher_impl.hpp"
+
+#include "gsl/span"
```

Comment:
this is external library, use <> instead ""

Reply:
fixed